src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/resource/ReportQuestionResponseResource.kt

```diff
@@ -174,12 +175,30 @@ class ReportQuestionResponseResource(
       ),
     ],
   )
-  fun deleteLastQuestionAndResponses(
+  fun deleteQuestionsAndResponses(
     @Schema(description = "The incident report id", example = "11111111-2222-3333-4444-555555555555", requiredMode = Schema.RequiredMode.REQUIRED)
     @PathVariable
     reportId: UUID,
+    @Parameter(
```

Comment:
So this is not in the request body? 🤔

Reply:
yeah query string as we discussed a couple of weeks ago